### ISSUE TITLE
aaa_base ln for mtab fails

### ISSUE THREAD

**Issue by bradfa** [bradfa/flaccid-linux#2]
Not sure why, but when installing the output looks like:

```
/ # installpkg --root /chroot/ /tmp/aaa_base-1-x86_64-1.tgz 
Verifying package aaa_base-1-x86_64-1.tgz.
Installing package aaa_base-1-x86_64-1.tgz:
PACKAGE DESCRIPTION:
WARNING:  Package has not been created with 'makepkg'
Executing install script for aaa_base-1-x86_64-1.tgz.
ln: mtab: File exists
Package aaa_base-1-x86_64-1.tgz installed.
```

**Comment by bradfa** [comment]
Also, it seems the /bin and /sbin symlinks are not being created.
Likely this is due to the `install/doinst.sh` using absolute paths starting in `/`.

**Comment by bradfa** [comment]
Fixed in 39dedd04fc7f3cc5719c0d818944fcfb8c1bffc0 by using relative paths.
